backends/platform/psp/png_loader.cpp

```diff
@@ -129,8 +129,9 @@ bool PngLoader::basicImageLoad() {
 	_channels = png_get_channels(_pngPtr, _infoPtr);
 
 	if (_colorType & PNG_COLOR_MASK_PALETTE) {
-		int paletteSize;
-		png_get_PLTE(_pngPtr, _infoPtr, nullptr, &paletteSize);
+		int paletteSize = 0;
+		png_color *palettePtr = nullptr;
+		assert(png_get_PLTE(_pngPtr, _infoPtr, &palettePtr, &paletteSize) == PNG_INFO_PLTE);
```

Comment:
Please don't call functions within `assert()`, as asserts are removed in release builds

Reply:
Done.